app/templates/users/edit.html

```diff
@@ -1,13 +1,57 @@
 {% extends "layouts/one-column.html" %}
+{% from "includes/forms.html" import checkbox_field, delete_button, fieldset %}
 
 {% set page_title = "User: " + user.username %}
+{% set breadcrumbs = [
+  {text: 'Home', url: '/'},
+  {text: 'Users', url: '/#Users'},
+  {text: 'User: ' + user.username}
+] %}
+{% set debug_items = [
+  {name: 'user', value: user},
+  {name: 'apps_options', value: apps_options},
+  {name: 'buckets_options', value: buckets_options}
+] %}
 
 {% block main_column %}
 
-<pre>user = {{ user | dump(4) | safe }}</pre>
-<br />
-<pre>apps_options = {{ apps_options | dump(4) | safe }}</pre>
-<br />
-<pre>buckets_options = {{ buckets_options | dump(4) | safe }}</pre>
+<form action="{{ url_for('users.edit', { id: user.auth0_id | urlencode }) }}" method="post">
 
+  {% call fieldset(
+    "superadmin",
+    "Super Admin",
+    hint="User is a super admin, allowing all privileges on the Control panel."
+  ) -%}
+    <div class="govuk-checkboxes">
+      {{ checkbox_field("superadmin", "Super Admin", user.is_superuser) }}
+    </div>
+  {%- endcall %}
+
+  <div class="form-group">
+    <input type="submit" class="button button-secondary" value="Save changes">
+  </div>
+
+</form>
+
+<form method="{{ url_for('users.reset_mfa', { id: user.auth0_id | urlencode }) }}" method="post">
```

Comment:
Hang on, shouldn't this be `action` instead of `method`?

Reply:
Yes, I thought I committed that fix already, sorry.

Reply:
Good catch, by the way - it took me about an hour to track that one down earlier 🤦‍♂️

Reply:
eheh you know by now that I notice the weirdest things 😆 - but yeah, there is a law of physics that says that the amount of time required for a fix is inversely proportional to the number of characters involved 🕳